styles_.css

```diff
@@ -0,0 +1,1069 @@
+    width: 53px;
+  }
+
+  .mylanguage-header-title::after {
+    content: url(img/logo.png);
+    position: absolute;
+    right: 10px;
+  }
+
+  .mylanguage-header-title-1::after {
+    content: url(./img/logo.png);
+    transform: rotate(90deg);
+    position: absolute;
+    right: 10px;
+  }
+
+  .myprofile-details {
+    display: grid;
+    grid-template-columns: 30% 70%;
+    flex-wrap: wrap;
+    grid-gap: 40px;
+    margin-top: 0;
+  }
+
+  .myprofile-details-container {
+    left: 20%;
+    display: block;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    width: 179%;
+  }
+
+  .mylanguages-place-holder {
+    width: 67%;
+    left: 13%;
+  }
+
+  .myprofile-details-header-title {
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    color: #091e42;
+    display: flex;
+    align-items: center;
+    margin-top: 0;
+  }
+
+  .myprofile-details-header {
+    width: 87%;
+  }
+
+  .myprofile-details-header-dsc {
+    width: 79%;
+    height: 20%;
+    top: 13%;
+  }
+
+  .myprofile-details-header-connect {
+    top: 0%;
+  }
+
+  .mylink-media {
+    top: 18%;
+    left: -6%;
+  }
+
+  .mylink-button-profile {
+    top: 11%;
+  }
+
+  .mylanguage-header-title .mylanguage-header-title-1::after {
+    content: url(img/logo.png);
+    transform: rotate(90deg);
+    position: absolute;
+    right: 0;
+  }
+
+  .mylanguage-keyfeatures-tech {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .mylanguage-tech-list-button {
+    top: 12px;
+    width: 122px;
+    height: 122px;
+    display: block;
+    position: relative;
+    background: #f7f7f9;
+    border-radius: 8px;
+    left: 12px;
+    margin-right: 10px;
+  }
+
+  hr {
+    width: 100%;
+    border-style: groove;
+    margin-left: 15px;
+    margin-bottom: 20px;
+  }
+
+  .mylanguage-frameworks-list .mylanguage-skills-list {
+    margin-top: 3%;
+    margin-bottom: 3%;
+  }
+
+  #logo {
+    display: none;
+  }
+
+  .mycontact {
+    background-image: url("img/backgdeskop.png");
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .mycontact-form-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .mycontact-form-header {
+    margin-top: 20px;
+    width: 100%;
+    position: relative;
+  }
+
+  .line-spacing {
+    padding: 5%;
+  }
+
+  .mycontact-form-header-dsc {
+    position: relative;
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
+
+  .mycontact-form {
+    position: inherit;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    left: 193px;
+  }
+
+  .mycontact-form-input {
+    width: 27.938rem;
+    height: 3rem;
+    margin-bottom: 20px;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .navbar {
+    align-self: center;
+    margin-right: 6%;
+  }
+
+  .mypage {
+    height: 90vh;
+  }
+
+  .mypage-container {
```

Comment:
Kindly ensure that the mobile version is responsive until the desktop screen size.

![image](https://user-images.githubusercontent.com/20554583/222940517-955026d0-6cf4-4406-872c-64e05ed32f32.png)

![image](https://user-images.githubusercontent.com/20554583/222940519-05f6060f-e485-498f-84a2-7631695e707d.png)